notifications_utils/clients/zendesk/zendesk_client.py

```diff
@@ -71,6 +75,10 @@ def send_ticket_to_zendesk(self, ticket):
 
         current_app.logger.info(f"Zendesk create ticket {ticket_id} succeeded")
 
+    def _is_user_suspended(self, response):
+        requester_error = response["details"].get("requester")
+        return requester_error and ("suspended" in requester_error[0]["description"])
```

Comment:
Should this check all of the requester_errors rather than just the first?

Or potentially this should only suppress the error if the only error is the user being suspended and we'd be interested to know about other errors?

Reply:
I couldn't find documentation about possible errors and what they would look like so wanted to only downgrade errors due to the user being suspended and not change what happens with anything else (we could then decide what to do with others if we came across them).

I have only ever seen errors due to someone being suspended take this format `{'requester': [{'description': 'Requester: Name is suspended.'}]}`. I think for anything else it might be useful to see the error.